packages/browsers/test/src/chrome/install.spec.ts

```diff
@@ -138,6 +138,7 @@ describe('Chrome install', () => {
   });
 
   it('falls back to the chrome-for-testing dashboard URLs if URL is not available', async function () {
+    this.timeout(60000);
```

Comment:
That addition is expected?

Reply:
Most likely yes, usually these these are slow as they do real request, and often fail on CI only because of timeouts.